snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -605,25 +610,29 @@ def _generate_build_env(self) -> str:
         if isinstance(self.plugin, plugins.v1.PluginV1):
             raise RuntimeError("PluginV1 not supported.")
 
-        # TODO add Snapcraft's environment.
+        # Snapcraft's say.
+        part_build_environment = get_part_build_environment(self)
```

Comment:
get_part_build_environment() feels misleading. 

Maybe use `get_snapcraft_part_environment()` and distinguish between "user" & "snapcraft"? or "common"?

Reply:
sure, will do that.